kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -117,13 +118,19 @@
       args: [
         "/usr/bin/tensorflow_model_server",
         "--port=9000",
+      ] +
+      if $.params.exposeRestApi then ["--rest_api_port=8000"] else []
+      + [
         "--model_name=" + $.params.modelName,
         "--model_base_path=" + $.params.modelPath,
       ],
       ports: [
         {
           containerPort: 9000,
         },
+        {
+          containerPort: 8000,
```

Comment:
should this be conditional?

Reply:
I've let this be there because it does not impact how the service behaves.

Reply:
is the port actually exposed though? I think one point of setting the port is to let people/services know which ports are open. So if the port isn't open we probably shouldn't list it.

Reply:
Also, if the http proxy is deployed, will it cause problem?